src/features/projects/components/project-nav-tabs.tsx

```
@@ -0,0 +1,42 @@
+        router.push(`/workspace/${workspaceId}/project/${projectId}/${segment}`);
+    };
+
+    return (
+        <div className="flex items-center border-b bg-background px-4 gap-1">
+            {TABS.map(({ label, icon: Icon, segment }) => {
+                const isActive = activeSegment === segment;
+                return (
+                    <button key={segment} onClick={() => navigate(segment)}
+                        className={cn("flex items-center gap-1.5 px-3 py-2.5 text-sm font-medium border-b-2 transition-colors",
+                            isActive ? "border-primary text-primary" : "border-transparent text-muted-foreground hover:text-foreground hover:border-muted-foreground"
+                        )}>
```

Biome (2.4.15)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)